types/iframe-resizer/iframe-resizer-tests.ts

```diff
@@ -1,4 +1,4 @@
-import { IFrameComponent, IFrameOptions, iframeResizer } from "iframe-resizer";
+import iframeResizer, { IFrameComponent, IFrameOptions } from "iframe-resizer";
```

Comment:
together with my other comment, `iframeResizer` should be imported as `import iframeResizer = require('iframe-resizer')`. That in turn means that IFrameComponent and the rest will need to be nested inside a namespace like so:

```ts
export namespace iframeResizer {
  interface IFrameObject { ...
```

and namespaced in the tests as `iframeResizer.IFrameComponent`.